ansible_base/authentication/common.py

```diff
@@ -101,16 +106,26 @@ def create_claims(authenticator: Authenticator, username: str, attrs: dict, grou
             is_superuser = has_permission
         elif auth_map.map_type == 'is_system_auditor':
             is_system_auditor = has_permission
-        else:
+        elif auth_map.map_type == 'team':
             if auth_map.organization not in org_team_mapping:
                 org_team_mapping[auth_map.organization] = {}
             org_team_mapping[auth_map.organization][auth_map.team] = has_permission
+        elif auth_map.map_type == 'organization':
+            org_membership[auth_map.organization] = has_permission
+        elif auth_map.map_type == 'role':
+            roles_mapping[auth_map.role] = has_permission
+        else:
+            logger.error(f"Map type {auth_map.map_type} of rule {auth_map.name} does not know how to be processed")
 
     return {
         "access_allowed": access_allowed,
         "is_superuser": is_superuser,
         "is_system_auditor": is_system_auditor,
-        "claims": org_team_mapping,
+        "claims": {
+            "team_membership": org_team_mapping,
+            "roles": roles_mapping,
+            "org_membership": org_membership,
```

Comment:
Nitpick: can we standardize on "org" or "organization"? The map type is "organization" but here it's "org" - I want to make it easy for API users to intuit which one to use.

Reply:
Anything user facing I try to do `organization` but internally I typically use `org`. That being said, we've talked about exposing the claims field in such a way that it could be user facing so I'll update this one too.